.github/workflows/schemas_release_pr.yml

```diff
@@ -0,0 +1,50 @@
+name: Schemas release PR
+
+on:
+  repository_dispatch:
+      types: schemas_release
+
+jobs:
+  python-dependencies:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          echo "::set-env name=PYTHON_VERSION::$(cat .python-version)"
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Install apt dependencies
+        run: sudo apt-get install libsnappy-dev libgconf-2-4
+      - name: Install Pipenv
+        run: pip install pipenv==2018.11.26
+      - name: Cache virtualenv
+        id: cache-virtualenv
+        uses: actions/cache@v1
+        with:
+          path: ~/.local/share/virtualenvs/
+          key: ${{ runner.os }}-${{ env.PYTHON_VERSION }}-virtualenvs-${{ hashFiles('Pipfile.lock') }}
+      - name: Install virtual environment
+        if: steps.cache-virtualenv.outputs.cache-hit != 'true'
+        run: pipenv install --dev
+  create-pull-request:
+    needs: python-dependencies
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: 'master'
+      - name: Update schemas version
+        run: ./scripts/update_schemas_version.sh
+      - run: |
+          echo "::set-env name=SCHEMAS_VERSION::$(cat .schemas-version)"
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          commit-message: Update schemas to version ${{ env.SCHEMAS_VERSION }}
+          title: Update schemas to version ${{ env.SCHEMAS_VERSION }}
+          body: This is an auto-generated PR with schema release update.
```

Comment:
I think we can drop the "with schema release update bit" as it's covered by the pull request title.